gamemode/modules/inventory/sv_inventory_interaction.lua

```diff
@@ -30,6 +38,7 @@ local function getSendableInventory( ent, inventory, isPlayerUpdate )
 
     for k, v in pairs( inventory.Inventory ) do
         v.item = table.Merge( v.item, RVR.Items.getItemData( v.item.type ) )
+        removeFunctions( v.item )
```

Comment:
Yeah, table.WriteTable throws an error when writing a function

Reply:
Theres a card up to not send the entire item data, and make the client grab the extra info.
This will mean we won't need `removeFunctions` anymore.
But thats tech debt we don't have time for :)